File: Data Structures & Algorithms/min-cost-climbing-stairs/submission-4.cpp
class Solution {
public:
    int minCostClimbingStairs(vector<int>& cs) {
        int dp[101];
        dp[1]=0;dp[2]=0;
        for(int i=2;i<=cs.size();++i){
            dp[i] = min(dp[i-1]+cs[i-1],dp[i-2]+cs[i-2]);
        }
        return dp[cs.size()];
    }
};

File: Data Structures & Algorithms/max-water-container/submission-4.py
class Solution:
    def maxArea(self, nums: List[int]) -> int:
        low, high = 0, len(nums)-1
        ans = 0
        while low <high:
            ans = max(ans, min(nums[low],nums[high])*(high-low))
            if nums[low] < nums[high]:
                low +=1
            else :
                high -=1
        return ans